Casks/w/wpsoffice.rb

```diff
@@ -2,8 +2,8 @@
   arch arm: "arm64", intel: "x64"
 
   version "6.10.2,0024.21300060"
-  sha256 arm:   "3a90be5d9fb48e188f47273a15f66276a20b3b8d53db85ea01f742740e7aae39",
-         intel: "0aa3c2b23f53ff8f56264b8a6dde601bfda4c9363a9760632a62d548a3c4d600"
+  sha256 arm:   "a613a851f967890226797eb755530ce120c4fd263b24e8b43c1d5d6c82fd2565",
+         intel: "0de6b2f7cbb751a13db9ea9632c2038d52fd829be7fd3643b14f0e90ed40e8b0"
```

Comment:
```suggestion
  sha256 :no_check
```

It seems that the binary is updated in place during releases, so we cannot reliably compute checksums.

Reply:
That should work

Reply:
Computing the checksum itself works, but we have to use `:no_check` since the binary is updated in-place. Otherwise, there will always be a time span between new upstream releases and cask version bumps, in which users encounter checksum errors.

Reply:
Understand. `no_check` is better choice.